cmd/auth-rest/templates/index.html

```diff
@@ -0,0 +1,157 @@
+<!--
```

Comment:
Shoudn't we have VueJS project for the UI ?

Reply:
You are right i should write that in vue. Will change the pr